testplan/web_ui/testing/public/index.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
```

Comment:
It throws a large annoying exception in the browser console when the page opens. [We stopped generating the manifest.json](https://github.com/Morgan-Stanley/testplan/blob/014ed0c9f8dd117d47ad0cf426286113ff78c310/testplan/web_ui/testing/src/index.js#L33) 11 months ago so this tag's been doing nothing except polluting browser consoles ever since.